.github/scripts/ci.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+WORKDIR=${PWD}
+
+# Getting the ARM toolchain
+wget https://developer.arm.com/-/media/Files/downloads/gnu-rm/9-2019q4/RC2.1/gcc-arm-none-eabi-9-2019-q4-major-x86_64-linux.tar.bz2
+tar -xf gcc-arm-none-eabi-9-2019-q4-major-x86_64-linux.tar.bz2
+export PATH=${PWD}/gcc-arm-none-eabi-9-2019-q4-major/bin:$PATH
+rm gcc-arm-none-eabi-9-2019-q4-major-x86_64-linux.tar.bz2
```

Comment:
you can unpack this on the fly with:
`wget -qO-  https://developer.arm.com/-/media/Files/downloads/gnu-rm/9-2019q4/RC2.1/gcc-arm-none-eabi-9-2019-q4-major-x86_64-linux.tar.bz2 | tar -xj`

Reply:
Right, done